t/venom/test_suites/security_event_random_mac/05_setup_packetfence.yml

```diff
@@ -0,0 +1,288 @@
+    headers:
+      "Authorization": "{{.get_login_token.result.token}}"
+      "Content-Type": "application/json"
+    assertions:
+      - result.statuscode ShouldEqual 201
+
+- name: assign_a_password_to_the_user
+  steps:
+  - type: http
+    method: POST
+    url: '{{.pfserver_webadmin_url}}/api/v1/user/iastigmate/password'
+    ignore_verify_ssl: true
+    body: >-
+     {
+       "quiet": true,
+       "valid_from": "2021-12-06 00:00:00",
+       "expiration": "{{.inline_lx_iastigmate.password.expiration}}",
+       "actions": [
+         {
+           "type": "set_role",
+           "value": "1"
+         },
+         {
+           "type": "set_access_duration",
+           "value": "1h"
+         }
+       ],
+       "pid_overwrite": 0,
+       "pid": "iastigmate",
+       "email": "iastigmate@test.ca",
+       "sponsor": "",
+       "password": "password",
+       "login_remaining": null,
+       "gender": "",
+       "title": "",
+       "firstname": "Igmout",
+       "lastname": "Astigmate",
+       "nickname": "",
+       "company": "",
+       "telephone": "",
+       "cell_phone": "",
+       "work_phone": "",
+       "address": "",
+       "apartment_number": "",
+       "building_number": "",
+       "room_number": "",
+       "anniversary": "",
+       "birthday": "",
+       "psk": "",
+       "notes": "",
+       "custom_field_1": "",
+       "custom_field_2": "",
+       "custom_field_3": "",
+       "custom_field_4": "",
+       "custom_field_5": "",
+       "custom_field_6": "",
+       "custom_field_7": "",
+       "custom_field_8": "",
+       "custom_field_9": "",
+       "access_duration": "1h",
+       "access_level": null,
+       "can_sponsor": null,
+       "category": "1",
+       "unregdate": null
+     }
+    headers:
+      "Authorization": "{{.get_login_token.result.token}}"
+      "Content-Type": "application/json"
+    assertions:
+      - result.statuscode ShouldEqual 201
+
+- name: configure_snat_interface_for_passthroughs
+  steps:
+  - type: http
+    method: PATCH
+    url: '{{.pfserver_webadmin_url}}/api/v1/config/base/network'
+    ignore_verify_ssl: true
+    body: >-
+      {
+        "interfaceSNAT":"eth0"
+      }
+    headers:
+      "Authorization": "{{.get_login_token.result.token}}"
+      "Content-Type": "application/json"
+    assertions:
+      - result.statuscode ShouldEqual 200
+      - result.bodyjson.message ShouldEqual "Settings updated"
+
+- name: configure_snat_interface_for_inline
+  steps:
+  - type: http
+    method: PATCH
+    url: '{{.pfserver_webadmin_url}}/api/v1/config/base/inline'
+    ignore_verify_ssl: true
+    body: >-
+     {
+       "id": "inline",
+       "interfaceSNAT": "eth0"
+     }
+    headers:
+      "Authorization": "{{.get_login_token.result.token}}"
+      "Content-Type": "application/json"
+    assertions:
+      - result.statuscode ShouldEqual 200
+
+- name: restart_iptables
+  steps:
+  - type: systemctl_service_restart
+    service: packetfence-iptables
```

Comment:
All services below will need to be restarted through API to be consistent with changes made on #7367